semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/bad1/error.txt

```diff
@@ -1,2 +1,14 @@
-rules/syntax/bad1.yaml: inside rule id eqeq-is-bad, pattern fields can't look like this: only ['pattern-inside', 'pattern-not-inside', 'pattern-either', 'pattern-not', 'pattern', 'patterns', 'pattern-where-python', 'fix', 'equivalences', 'pattern-regex'] operators can have children, but found `['pattern-inside']` with children
+error: pattern-inside cannot have children
+  --> rules/syntax/bad1.yaml:4
+4 |       - pattern-inside:
+5 |         - pattern: $X == $X
+6 |         - pattern: $X != $X
+7 |         - patterns:
+8 |           - pattern-inside: |
+9 |               def __init__(...):
+10 |                   ...
```

Comment:
Can we align the `|`

Reply:
yeah, I had put it off for v0 but it isn't hard to fix.

Reply:
fixed.